docker/docker-compose.yml

```diff
@@ -7,40 +7,40 @@ services:
     command: ["--quiet", "--logpath", "/dev/null"]
     volumes:
       - ./data/chronicle/mongodb:/data/db
-    environment:
-      - MONGO_INITDB_ROOT_USERNAME=${MONGODB_USERNAME}
-      - MONGO_INITDB_ROOT_PASSWORD=${MONGODB_PASSWORD}
+    # environment:
+    #   - MONGO_INITDB_ROOT_USERNAME=${MONGODB_USERNAME}
+    #   - MONGO_INITDB_ROOT_PASSWORD=${MONGODB_PASSWORD}
     ports:
       - 27017:27017
 
-  inx-chronicle:
-    container_name: inx-chronicle
-    depends_on:
-      influx:
-        condition: service_started
-      hornet:
-        condition: service_healthy
-    build:
-      context: ..
-      dockerfile: docker/Dockerfile.debug
-    image: inx-chronicle:dev
-    ports:
-      - "8042:8042/tcp" # REST API
-      - "9100:9100/tcp" # Metrics
-    tty: true
-    deploy:
-      restart_policy:
-        condition: on-failure
-        delay: 5s
-        max_attempts: 3
-    command:
-      - "--mongodb-conn-str=${MONGODB_CONN_STR}"
-      - "--influxdb-url=http://influx:8086"
-      - "--influxdb-username=${INFLUXDB_USERNAME}"
-      - "--influxdb-password=${INFLUXDB_PASSWORD}"
-      - "--inx-url=http://hornet:9029"
-      - "--jwt-password=${JWT_PASSWORD}"
-      - "--jwt-salt=${JWT_SALT}"
+  # inx-chronicle:
```

Comment:
you can create a copy of docker-compose.yml and use it with `docker-compose -f docker-compose-copy.yml`. This way you don't need to uncomment before push.